api/apps/api/src/modules/projects/dto/update-project-blm-range.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsArray } from 'class-validator';
+import { IsValidRange } from '@marxan-api/decorators/is-valid-range.decorator';
+
+export class UpdateProjectBlmRangeDTO {
+  @ApiProperty({ example: [0.001, 100] })
+  @IsArray()
+  @IsValidRange()
```

Comment:
Isn't it already validated in previous PR ?

Reply:
Yes, it might be a bit overkill to validate it at controller and application levels. As I implemented the application code first, I didn't have this validation yet, so when implementing this endpoint I thought this would be better. Then, before removing the next validation I thought "well, a double check won't hurt anybody", but giving it a second thought now, if we change the validation details of a range we have two places to do so now.
So, we can remove the application level validation, or we could also extract the validation rules to a utils where both validations check the range.

Reply:
ok, I see this now what we were discussing in DM 😅 

My suggestion would be to remove the application level validation and to *not* extract the validation rules - actually extracting them was my first reaction when I looked at the decorator's logic, but if we don't actually need to validate downstream (=if we don't currently expect these ranges to come in through any other route than controllers using this DTO), then I would not bother. If we do need to account for other data ingress routes at some point, I'd definitely recommend to extract the logic at that point and share it between ingress routes as needed.

Reply:
Perfect, I deleted the inner validation.

Reply:
Sorry to reopen.

@hotzevzl @AaronPerezPerez as this is our kind of valid data rule, I think its better to have it "inner", as not every single update request will come through REST API.

Reply:
Okeey, restoring it!